docs/data-configuration.md

```diff
@@ -109,6 +109,19 @@ _Optional_: This can be used to convert any indicator IDs in the data documentat
 docs_indicator_url: https://my-github-org.github.io/my-site-repository/[id]
 ```
 
+### docs_metadata_fields
+
+_Optional_: This can be used to configure the metadata report on the SDG data service site, to add metadata fields to the report you must add the key and label for each field to you wish to display. Here is an example:
```

Comment:
@Govej sorry I didn't notice previously by can you start new sentence after "data service site". Also can you swap "data service site" to "data documentation website", just realised that's what's used for similar fields so will keep it consistent

Reply:
@LucyGwilliamAdmin Updated